iothub/device/src/Transport/Mqtt/MqttTransportHandler.cs

```diff
@@ -333,28 +337,25 @@ public override async Task<Message> ReceiveAsync(TimeoutHelper timeoutHelper)
             }
         }
 
-        private Message ProcessMessage(bool hasMessage)
+        private Message ProcessMessage()
         {
             Message message = null;
 
             try
             {
                 if (Logging.IsEnabled)
-                    Logging.Enter(this, message, $"hasMessage={hasMessage}", nameof(ProcessMessage));
+                    Logging.Enter(this, message, $"Will begin processing received C2D message, queue size={_messageQueue.Count}", nameof(ProcessMessage));
```

Comment:
cloud-to-device

Reply:
this is an internal log, so I don't actually mind leaving it as is